DoSomething/tower
# Ansible Inventory File

Initial commit - synced from jenkins box.

DoSomething/devops#251

#### What's this PR do?
#### Where should the reviewer start?

Please review for decomissioned and missing hosts.

#### How should this be manually tested?

```
ansible --vault-password-file=.vault.txt -i production Thor -m ping --list-hosts
  hosts (10):
    thor-web1
    thor-northstar-web
    thor-northstar-mongo
    thor-varnish
    thor-mongo-mb
    thor-solr
    thor-rabbit
    mjolnir
    thor-haproxy
    thor-mbc
```

Run without `--list-hosts`  to check keys/user setup.
